Cargo.toml

```diff
@@ -16,7 +16,7 @@ exclude     = [
 ]
 
 [dependencies]
-libc = { git = "https://github.com/rust-lang/libc" }
+libc = { version = "0.2.57", features = ["extra_traits"] }
```

Comment:
Don't switch libc from a git dependency to a version dependency.  We only do that at release time.  Also, we'll probably merge #1035 first, so you won't have to change this file.